.docker/release/Dockerfile

```diff
@@ -1,7 +1,7 @@
-# moveit/moveit:melodic-release
+# moveit/moveit:crystal-release
 # Full debian-based install of MoveIt! using apt-get
 
-FROM ros:melodic-ros-base
+FROM ros:crystal-ros-base-bionic
```

Comment:
I guess this file is not yet completely migrated - it has so few changes only. 
If so, I suggest to remove it too or have a TODO.

Reply:
Sure thing